### DISCUSSION TITLE
Source Data Path

### DISCUSSION THREAD

**Discussion by aravinth-numberz** [KruxAI/ragbuilder#7] [Q&A] [answered]
I have given the source data as
`~/Users/aravinth/Documents/RAGdocs`
But I'm getting `Invalid source data. Please check the URL or file/ directory path.` as the error popup.
What I'm doing wrong here ?

**Comment by aravind10x** [comment, marked as answer]
Hi @aravinth-numberz , Sorry for the late reply! 
Will get fixed with this PR: https://github.com/KruxAI/ragbuilder/pull/14

**Reply by aravinth-numberz** [reply]
No issues, Thanks for your reply @aravind10x

**Comment by DefamationStation** [comment]
Hi is this not fixed yet? I am getting the same issue on the latest built with docker

![image](https://github.com/user-attachments/assets/9a2e5692-c60a-4b48-a8d0-e0bd2ee86a52)

![image](https://github.com/user-attachments/assets/2cc661db-1cbc-46eb-b778-0d50c662e791)
